README.md

```diff
@@ -4,4 +4,9 @@
 ### Oversight
 ### Decision Making, Conflict Resolution, and Facilitation
 ### [Proposing projects – the charter process](charters/README.md)
-### [Proposing enhancements – the RFC process](rfcs/text/0001-rfc-process.md)
\ No newline at end of file
+### [Proposing enhancements – the RFC process](rfcs/text/0001-rfc-process.md)
+
+## DCP Strategy
```

Comment:
We should think about what content we want to automatically populate on https://data.humancellatlas.org from this community site.